src/components/Approval/Approval.js

```diff
@@ -64,64 +62,38 @@ const Approval = ({
 
   return (
     <>
-      {assetNumber === '1' && (
```

Comment:
Remove the allowance state check; it doesnt really serve a purpose. We only need to know if approval *IS* required, not if it *IS NOT*